### DISCUSSION TITLE
Snapshot WRT1900ACS

### DISCUSSION THREAD

**Discussion by vector-co-uz** [immortalwrt/immortalwrt#868] [Q&A] [answered]
I'm using firmware builder of Immortalwrt, my device is Linksys WRT1900acs v2.
when I flash a snapshot, the router does not respond, does not give out the IP address.
with the stable version 21.02.3 everything is fine.
even without the firmware builder, the snapshot version doesn't work on my router.

**Comment by 1715173329** [comment]
do you mean dhcp doesn't work? Have you tried setting a fixed ip addr & gateway? If it doesn't boot, then you need catch the TTL log

**Reply by vector-co-uz** [reply]
> do you mean dhcp doesn't work? Have you tried setting a fixed ip addr & gateway? If it doesn't boot, then you need catch the TTL log

At first I also thought that DHCP was not working, but then I found out that the router was hanging in one place.
I would like to disassemble the case for TTL. But let me do it anyway.

**Comment by vector-co-uz** [comment]
**TTL log is here:**

`Watchdog handover: fd=3
- watchdog -
killall: telnetd: no process killed
killall: ash: no process killed
Thu Feb  9 05:37:43 UTC 2023 upgrade: Sending TERM to remaining processes ... ub urngd logd rpcd hostapd[  180.765684] device wlan1 left promiscuous mode
 wpa_supplicant[  180.770406] br-lan: port 5(wlan1) entered disabled state
 wpa_supplicant hostapd netifd uhttpd dbus-daemon[3625]: <warn>  could not acquire the 'org.freedesktop.ModemManager1' service name
[3625]: <info>  ModemManager is shut down
 sh ntpd dnsmasq
[  180.965207] device wlan0 left promiscuous mode
[  180.969711] br-lan: port 6(wlan0) entered disabled state
Thu Feb  9 05:37:46 UTC 2023 upgrade: Sending KILL to remaining processes ...
[  184.869141] sh (5531): drop_caches: 3
Thu Feb  9 05:37:47 UTC 2023 upgrade: Switching to ramdisk...
[  186.061964] UBIFS (ubi0:1): background thread "ubifs_bgt0_1" stops
[  186.072989] UBIFS (ubi0:1): un-mount UBI device 0
Thu Feb  9 05:37:48 UTC 2023 upgrade: Performing system upgrade...
Unlocking kernel2 ...

Writing from <stdin> to kernel2 ...
Thu Feb  9 05:37:53 UTC 2023 upgrade: Upgrade completed
Thu Feb  9 05:37:54 UTC 2023 upg[  191.871440] UBIFS (ubi1:0): un-mount UBI device 1
rade: Rebooting [  191.877610] UBIFS (ubi1:0): background thread "ubifs_bgt1_0" stops
system...
umount: can't unmount /dev: Resource [  191.888615] reboot: Restarting system

0

NAND read: device 0 offset 0x3200000, size 0x600000
 6291456 bytes read: OK
## Booting kernel from Legacy Image at 02000000 ...
   Image Name:   ARM ImmortalWrt Linux-5.15.91
   Created:      2023-02-05  20:24:46 UTC
   Image Type:   ARM Linux Kernel Image (uncompressed)
   Data Size:    3700698 Bytes = 3.5 MiB
   Load Address: 00008000
   Entry Point:  00008000
   Verifying Checksum ... OK
   Loading Kernel Image ... OK
OK

Starting kernel ...`

**Reply by vector-co-uz** [reply]
Used official Immortalwrt snapshot.

**Comment by 1715173329** [comment, marked as answer]
Okay, it should be fixed by https://github.com/openwrt/openwrt/commit/721206a1b14fc5eba7ba8e20d979b26baf934e0d. I will update our  prebuilt images too.

**Reply by vector-co-uz** [reply]
Fixed?

**Reply by 1715173329** [reply]
yup
